packages/ngx-forms/src/lib/upload/components/upload-zone/upload-zone.component.html

```diff
@@ -1,10 +1,16 @@
 <div *ngIf="uploader.options.type === 'drop'" [class.is-disabled]="disabled" class="m-upload">
+  <small class="m-upload__description u-margin-bottom-xs" id="description">
```

Comment:
The id should be definable or generated with a random string (like in other components, e.g. time picker), in case there are more than one upload zones on a page.